plugins/mimecast/plugin.spec.yaml

```diff
@@ -1460,8 +1474,34 @@ actions:
         description: Event logs data
         type: "[]audit_events_response"
         required: true
+        example: []
       pagination:
         title: Pagination
         description: Pagination for request
         type: "pagination"
-        required: false
\ No newline at end of file
+        required: false
+        example: {}
+tasks:
+  monitor_siem_logs:
+    title: Monitor SIEM LOGS
+    description: Monitor and retrieve the latest logs
+    input:
+      token:
+        title: Token
+        description: Used to request the next available log file
+        type: string
+        required: false
+        example: 9de5069c5afe602b2ea0a04b66beb2c0
+    output:
+      data:
+        title: Data
+        description: List of logs
+        type: "[]object"
+        required: true
+        example: []
+      datetime:
```

Comment:
There should only be one output here, for the data array. Datetime feels like something you may want to be tracking in the state instead.